### PR TITLE
LPD-39385 Wildcard * does not work for URL redirection

### PR DESCRIPTION
**Pull request by MrJohn1911**

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:site-management

**Comment by jonathanmccann** [comment]
ci:report:44658863

File: portal-impl/src/com/liferay/portal/struts/FindStrutsAction.java [hunk 1]
```diff
@@ -219,16 +219,36 @@ private String _getNoSuchEntryRedirect(
...
+						allowedDomain.length() - 1) ||
+					 allowedDomain.regionMatches(
+						 2, domain, 0, domain.length()))) {
+
+					return noSuchEntryRedirect;
+				}
+			}
+		}
+		else {
+			if (ArrayUtil.contains(
```
Comment (jonathanmccann) [inline portal-impl/src/com/liferay/portal/struts/FindStrutsAction.java:246]: This `else` and `if` can probably be combined.

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:report:46467708

**Comment by MrJohn1911** [comment]
@jonathanmccann no regressions found!

**Comment by jonathanmccann** [comment]
ci:forward

**Comment by MrJohn1911** [comment]
ci:forward

**Comment by MrJohn1911** [comment]
ci:forward:force

**Pull request closed without merging**